examples/spe1/stages.yml

```diff
@@ -0,0 +1,32 @@
+-
+  name: simulate_SPE1
+
+  input:
+    -
+      record: field_properties
+      location: field_properties.json
+
+  output:
+    -
+      record: summary
+      location: summary.json
+
+  transportable_commands:
+    -
+      name: bash
+      location: bash.py
+    -
+      name: summary2json
+      location: summary2json.py
+    -
+      name: render
+      location: render.py
+
+  script:
+    # TODO: The following cp should not be present, instead files should be
+    # transfered as opaque data
+    - bash cp ${SPE1_WORKSPACE_ROOT}/resources/SPE1CASE2.DATA.jinja2 SPE1CASE2.DATA.jinja2
+    - bash echo "{}" > parameters.json
```

Comment:
Because `template_render` from `libres` currently assumes there to be a `parameters.json` file that it can load `GEN_KW` values from. We should probably loosen that requirement at some point...

Until then we get a nice demo of the `bash` command 🚀